pkg/serviceaccount/legacy.go

```diff
@@ -36,6 +36,8 @@ import (
 	"k8s.io/klog/v2"
 )
 
+const invalidSinceLabelKey = "kubernetes.io/legacy-token-invalid-since"
```

Comment:
avoid redefining the string constant? define it in one place and share it?

Reply:
Define the InvalidSinceLabelKey in [pkg/serviceaccount/legacy.go](https://github.com/kubernetes/kubernetes/pull/120682/files/f061de074f0114d14ca5b575f9c0bde7a52da91e#diff-8d61d5a32b0d241cb53f579ef1da6e4ae4e0b866e6a7a8205f98eb901c23128a) now and thus it can be shared.